psse/psse-converter/src/main/java/com/powsybl/psse/converter/PsseImporter.java

```diff
@@ -256,29 +367,44 @@ private static void createBuses(PsseRawModel psseModel, ContainersMapping contai
             // create bus
             createBus(psseBus, voltageLevel);
 
+            busNumToPsseBus.put(psseBus.getI(), psseBus);
+
         }
     }
 
     private static void createLine(PsseNonTransformerBranch psseLine, ContainersMapping containerMapping, PerUnitContext perUnitContext, Network network) {
         String id = "L-" + psseLine.getI() + "-" + psseLine.getJ() + "-" + psseLine.getCkt();
+
+        //build a unique name
+        String idTmp = id;
+        int i = 2;
+        while (network.getLine(idTmp) != null) {
+            idTmp = id + "-" + i;
+            i++;
+        }
+
+        id = idTmp;
```

Comment:
I think this should be moved in a utility function.
```
String getUniqueId(String id, Predicate<String> predicate) {
    String tmpId = id;
    int i = 2;
    while (predicate.test(tmpId)) {
        tmpId = id + "-" + i;
        i++
    }
}
```

Then use this method everywhere:
```
String id = "L-" + psseLine.getI() + "-" + psseLine.getJ() + "-" + psseLine.getCkt();
id = getUniqueId(id, s -> network.getLine(s) != null);
```

Reply:
OK, proposition of code done